server/server.go

```diff
@@ -201,7 +201,7 @@ func (s *Server) onConn(c net.Conn) {
 	}
 
 	s.rwlock.Lock()
-	s.clients[conn.connectionID] = conn
+	s.clients[conn.connectionID] = conn // TODO:why unreuse?
```

Comment:
This line of code change is not related to this PR, you may open an issue for it.